### DISCUSSION TITLE
PROBLEM Controller - IKEA E2001/E2002 STYRBAR Remote control Hold Button

### DISCUSSION THREAD

**Discussion by 4lexjames** [EPMatt/awesome-ha-blueprints#476] [Q&A]
Hi guys, I am using the above blueprint, its all working but the hold button down for dimming isnt working, I have to release and press the button again. I have enabled loop until release but it still doesnt work, im using MQTT.

**Comment by AHoran** [comment]
Have a look in this PR https://github.com/EPMatt/awesome-ha-blueprints/pull/411

**Comment by martialllaw** [comment]
I'm also having issues with the Controller/Hook (latest version), My Rodret will increase brightness as expected on long press but dimming long press results in the light dimming all the way to zero no matter when the button is released.

**Comment by yarafie** [comment]
Can you try my fork

https://github.com/yarafie/awesome-ha-blueprints/tree/main
